lib/google-civic/request.rb

```diff
@@ -2,8 +2,8 @@
 
 module GoogleCivic
   module Request
-    def get(path,params={})
-      response = connection.get do |request|
+    def get(path,params={},connection_opts={})
+      response = connection(connection_opts).get do |request|
         request.url path
         params.each { |name, value| request.params[name] = value }
         request.params['key'] = @key
```

Comment:
Not sure what's you saw wrong with the indentation in this file, it was actually correct.